src/themes/uswitch/theme.json

```diff
@@ -505,6 +505,57 @@
     }
   },
 
+  "cta2": {
```

Comment:
why do we have a cta2?

Reply:
because it's a breaking change in the schema (like we talked about in the trustyle workshop last week). it's a stopgap until we have a theme builder